### PR TITLE
testing/cups-epson-escpr: new aport for cups driver

### PR DESCRIPTION
**Pull request by micw**
http://download.ebz.epson.net/dsc/search/01/search/?OSC=LX
Epson Inkjet Printer Driver (ESC/P-R) for Linux

This driver supports many epson inkjet printers

### CODE REVIEW

File: testing/cups-epson-escpr/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# $Id$
+# Maintainer: Michael Wyraz <michael@wyraz.de>
+# Adapated from https://aur.archlinux.org/packages/epson-inkjet-printer-escpr/
+#
+
+pkgname=cups-epson-escpr
+pkgver=1.6.29
+pkgrel=1
```
Comment (Ikke) [inline testing/cups-epson-escpr/APKBUILD]: pkgrel in alpine usually starts at 0

File: testing/cups-epson-escpr/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+pkgdesc="Epson Inkjet Printer Driver (ESC/P-R) for Linux"
+arch="all"
+url="http://download.ebz.epson.net/dsc/search/01/search/?OSC=LX"
+license="GPL2"
+depends="cups ghostscript"
+makedepends="cups-dev"
+options="!libtool"
+source="https://download3.ebz.epson.net/dsc/f/03/00/08/01/30/6cd80f9555b252d5d7101d450bb43a6a6e7f9689/epson-inkjet-printer-escpr-1.6.29-1lsb3.2.tar.gz build_x86_64.patch"
```
Comment (Ikke) [inline testing/cups-epson-escpr/APKBUILD]: It's customary to have additional source declarations on a new line, indented with a tab.

**Comment by Ikke** [comment]
If there is no test suite present (which I suspect is the case), you'll have to add !check to the options as well (with a comment that the test suite is missing).

**Comment by Ikke** [comment]
@micw A lot of projects ship with a test suite, usually unit tests, can also be some other kind of tests. If it's present, we want to execute that test suite in the check phase.

File: testing/cups-epson-escpr/APKBUILD [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+# Adapated from https://aur.archlinux.org/packages/epson-inkjet-printer-escpr/
+#
+
+pkgname=cups-epson-escpr
+pkgver=1.6.29
+pkgrel=0
+pkgdesc="Epson Inkjet Printer Driver (ESC/P-R) for Linux"
+arch="all"
+url="http://download.ebz.epson.net/dsc/search/01/search/?OSC=LX"
+license="GPL2"
```
Comment (rnalrd) [inline testing/cups-epson-escpr/APKBUILD]: Please use a valid license ID from https://spdx.org/licenses/

**Comment by rnalrd** [comment]
Please change the commit msg to
<pre>
testing/cups-epson-escpr: new aport

CUPS driver for Epson inkjet printers
</pre>

See https://wiki.alpinelinux.org/wiki/Creating_patches

Thanks!

**Comment by Ikke** [comment]
@micw Can you rebase this on the latest master? Then you will get drone CI integration and the PR will no longer be marked as stale.

**Comment by micw** [comment]
Sure. I rebased it and also altered the commit comments and license id. I also squashed the 3 commits into one.

**Comment by Ikke** [comment]
The source package seems to have disappeared

**Comment by micw** [comment]
@ikke thank you. For some reason I got no notification from CI about the failure :-(

File: testing/cups-epson-escpr/APKBUILD [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+pkgname=cups-epson-escpr
+pkgver=1.6.29
+pkgrel=0
+pkgdesc="Epson Inkjet Printer Driver (ESC/P-R) for Linux"
+arch="all"
+url="http://download.ebz.epson.net/dsc/search/01/search/?OSC=LX"
+license="GPL-2.0-or-later"
+depends="cups ghostscript"
+makedepends="cups-dev"
+options="!libtool !check" # Test suite is missing
```
Comment (maxwell-k) [inline testing/cups-epson-escpr/APKBUILD]: !libtool is the default setting so isn't required

```suggestion
options="!check" # Test suite is missing
```

File: testing/cups-epson-escpr/APKBUILD [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+source="https://sources.voidlinux.org/epson-inkjet-printer-escpr-1.6.29/epson-inkjet-printer-escpr-1.6.29-1lsb3.2.tar.gz
```
Comment (tcely) [inline testing/cups-epson-escpr/APKBUILD]:
```suggestion
source="https://sources.voidlinux.org/epson-inkjet-printer-escpr-$pkgver/epson-inkjet-printer-escpr-$pkgver-1lsb3.2.tar.gz
```

**Pull request closed without merging**